assets/js/modules/analytics-4/datastore/conversion-events.js

```diff
@@ -83,16 +62,11 @@ const baseSelectors = {
 	 *
 	 * @since 1.96.0
```

Comment:
Let's add a new `@since` here about the signature change.

Reply:
Updated.